server/src/test/kotlin/suwayomi/tachidesk/manga/controller/CategoryControllerTest.kt

```diff
@@ -0,0 +1,48 @@
+package suwayomi.tachidesk.manga.controller
+
+import org.jetbrains.exposed.sql.deleteAll
+import org.jetbrains.exposed.sql.transactions.transaction
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import suwayomi.BASE_PATH
+import suwayomi.tachidesk.manga.impl.Category
+import suwayomi.tachidesk.manga.model.table.CategoryTable
+import suwayomi.tachidesk.server.applicationSetup
+import xyz.nulldev.ts.config.CONFIG_PREFIX
+import java.io.File
+
+internal class CategoryControllerTest {
+
+    @BeforeEach
+    internal fun setUp() {
+        val dataRoot = File(BASE_PATH).absolutePath
+        System.setProperty("$CONFIG_PREFIX.server.rootDir", dataRoot)
+        applicationSetup()
+    }
+
+    @Test
+    fun categoryReorder() {
+        Category.createCategory("foo")
+        Category.createCategory("bar")
+        val cats = Category.getCategoryList()
+        val foo = cats.stream().filter { c -> c.name == "foo" }.findFirst().orElseThrow()
```

Comment:
Do this in a kotlin way(avoid streams) 😉 
```suggestion
        val foo = cats.asSequence().filter { it.name == "foo" }.first()
```

Reply:
Thanks for the hint :)